### DISCUSSION TITLE
new version timeline?

### DISCUSSION THREAD

**Discussion by etipton** [hairyhenderson/gomplate#1218] [Q&A] [answered]
Unless I'm mistaken, it looks like v3.9.0 is the latest official release. Any idea when the next release will be deployed?

I'm looking to use `{{aws.Account}}` -- but with SSO support -- and it appears (based on my poking around) that this will be possible in the next version (due to updated go AWS SDK). Just curious, thanks! ❤️  gomplate :)

**Comment by hairyhenderson** [comment, marked as answer]
> Unless I'm mistaken, it looks like v3.9.0 is the latest official release. Any idea when the next release will be deployed?

@etipton _imminently_ - I'm actually working on releasing 3.10.0 today 😉

**Reply by etipton** [reply]
!! good timing! 👍
